student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(age, classroom, name = 'Uknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.students.push(self)
```

Comment:
- [ ] As stated in the project requirement when you Create the has-many/belongs-to relationship between Classroom and Student, Kindly make sure  that when setting the classroom for a student it also adds it to the classrooms' students use `method that serves as the 'belongs-to' side (a student belongs to a classroom).` in the below sample because `classroom.students.push(self) unless classroom.students.include?(self)`: This line adds the current object (self) to the students array of the classroom object, but only if it’s not already included. This establishes a two-way relationship: not only does the student belong to a classroom (as indicated by @classroom), but the classroom also has this student in its list of students.So Kindly Adjust like this :

```ruby
 attr_reader :classroom
  .
  .
  # *Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom). Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```